### DISCUSSION TITLE
Error zipping multiple big files

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [ouch-org/ouch#353] [Q&A] [answered]
I've having an issue when trying to zip multiple big files at once although I feel like this is something weird with Windows rather than the crate hence why I didn't open an issue, just making sure.

I get `Large file option has not been set`.

I can't see anything about this option on either the repository itself or the docs. The same command worked fine when I compressed the files into a `.tar` file.

If this is indeed unrelated to the crate, it might be worth mentioning in your README!

**Comment by figsoda** [comment, marked as answer]
This seems like a bug, I've opened https://github.com/ouch-org/ouch/pull/354 to fix this

**Comment by AntoniosBarotsis** [comment]
Glad I mentioned it in that case :)

**Reply by marcospb19** [reply]
Yeah thanks for reporting! It was fixed in https://github.com/ouch-org/ouch/pull/354 by @figsoda.
Should be working as expected now.
